Tests/Serilog.Exceptions.Test/Filters/CompositeExceptionPropertyFilterTest.cs

```diff
@@ -19,65 +19,54 @@ public class CompositeExceptionPropertyFilterTest
         [Fact]
         public void CreationOfCompositeFilter_ForOneNullFilter_Throws()
         {
-            // Act
             var ex = Assert.Throws<ArgumentException>(
-                () => new CompositeExceptionPropertyFilter(new IExceptionPropertyFilter[] { null }));
+                () => new CompositeExceptionPropertyFilter(new IExceptionPropertyFilter[] { null! }));
 
-            // Assert
             Assert.Contains("index 0", ex.Message, StringComparison.Ordinal);
         }
 
         [Fact]
         public void ShouldFilterTheProperty_WhenFirstFilterFilters_Filters()
         {
-            // Arrange
             var filterA = new IgnorePropertyByNameExceptionFilter("A");
```

Comment:
Nitpicking: If you want to remove AAA annotations, let's do it consistently everywhere in other PR.

Reply:
Yeah, fair enough. Will add back in.

Reply:
Fixed